drivers/mdio/Kconfig.nxp_s32_gmac

```diff
@@ -0,0 +1,21 @@
+# Copyright 2023 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig MDIO_NXP_S32_GMAC
+	bool "NXP S32 GMAC MDIO driver"
+	default y
+	depends on DT_HAS_NXP_S32_GMAC_MDIO_ENABLED
+	depends on CLOCK_CONTROL
+	select NOCACHE_MEMORY
```

Comment:
Just a minor comment: `select NOCACHE_MEMORY if ARCH_HAS_NOCACHE_MEMORY_SUPPORT`

Reply:
right, and line above to `select CLOCK_CONTROL`